src/main/java/org/jabref/logic/bst/BstVM.java

```diff
@@ -93,7 +93,7 @@ public String render(Collection<BibEntry> bibEntries) {
         return render(bibEntries, null);
     }
 
-    protected Stack<Object> getStack() {
+    protected Deque<Object> getStack() {
```

Comment:
Please revert this modernizer issue here, stack das

Reply:
Tests run through. Is there a special reason? I did not find the PR where you tried.

Reply:
Note thar I needed to use LinkedList as implementation to allow for nulls in the stack...